recipes/ChessAnalysisPipeline/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "ChessAnalysisPipeline" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8edc4e86bf0760bc3d7ae4d46efd39ca25c2295baa3cc34e00d78d29329677b2
+
+build:
+  number: 0
+  noarch: python
```

It ha scripts and cannot be noarch.

```suggestion
  skip: true  # [py<38]
```